### PR TITLE
Reviewing Lines Part 3

### PR DESCRIPTION
**Pull request by ttgmichael**
Dolar signs on QUESTLV100 lines 1000-1300 with some revisions.

Let me know your thoughts! :)

### CODE REVIEW

File: QUEST_LV_0100.tsv [hunk 1]
```diff
@@ -1033,271 +1033,271 @@ QUEST_LV_0100_20150317_001032	$I should have requested this of you all at once.{
...
+QUEST_LV_0100_20150317_001144	{memo X}I can't stand seeing the Panto totems pressed with evil forces. {nl}I'm going to break it through the shaman doll. Want to give it a try?
+QUEST_LV_0100_20150317_001145	$We can't do anything until Basil restores the shaman doll perfectly. {nl}Unless for special situations like this, we can use, um, nothing.
+QUEST_LV_0100_20150317_001146	$It would have been a disaster if the Wild Carnivore had legs.{nl}I'm glad it was killed beforehand.
 QUEST_LV_0100_20150317_001147	{memo X}I heard we promised the first Paladin three things in return for saving our family. {nl}
-QUEST_LV_0100_20150317_001148	$To help build the Tenet Church, {nl} to protect the divine relics by having Watchers guard the sacred land, {nl} and lastly, to protect each other together when great danger arises. {nl}
-QUEST_LV_0100_20150317_001149	I do not know what curse it was. {nl} There's got to be a record within the Tenet Church.
+QUEST_LV_0100_20150317_001148	$To help build the Tenet Church, {nl}to appoint the Watchers to safeguard the sacred land covering the Holy Relic, {nl}and lastly, to protect each other together when great danger arises. {nl}
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1148]: In this PR https://github.com/Treeofsavior/EnglishTranslation/pull/1405/files, you named "divine object" to "sacred relic" and you're using the term 'Holy Relic' in this PR.

File: QUEST_LV_0100.tsv [hunk 2]
```diff
@@ -1033,271 +1033,271 @@ QUEST_LV_0100_20150317_001032	$I should have requested this of you all at once.{
...
+QUEST_LV_0100_20150317_001194	$If you ask me what the strongest weapon is, my answer would be my faith. Faith to an ally that {nl}you can truly rely on.
 QUEST_LV_0100_20150317_001195	$A Thaumaturge does not get stronger just by learning.{nl}You should experience it yourself.
 QUEST_LV_0100_20150317_001196	{memo X}
-QUEST_LV_0100_20150317_001197	Stealth, detection, trap installation. {nl} Scouts are masters of searching and monitoring.
+QUEST_LV_0100_20150317_001197	$Stealth, detection, trap installation. {nl}Scouts are masters of search and observation.
 QUEST_LV_0100_20150317_001198	$We Sadhus are always on an eternal journey. {nl}When our journeys in this world come to an end, we continue it, in the afterlife.
-QUEST_LV_0100_20150317_001199	Nothing is unfair in real battles.{nl}You've gotta do anything to win. Don't you think so?
-QUEST_LV_0100_20150317_001200	We should find it quickly..{nl}It may be an end to Mage Tower.
+QUEST_LV_0100_20150317_001199	$Nothing is unfair in real battles.{nl}You've gotta do anything to win. Don't you think so?
+QUEST_LV_0100_20150317_001200	$We should find it quickly..{nl}It may be an end to the Mage Tower.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1200]: ellipsis

File: QUEST_LV_0100.tsv [hunk 3]
```diff
@@ -1033,271 +1033,271 @@ QUEST_LV_0100_20150317_001032	$I should have requested this of you all at once.{
...
 QUEST_LV_0100_20150317_001229	$Thank you very much for saved us.{nl}
-QUEST_LV_0100_20150317_001230	$If you still want to enter the Closed Area, collect Vubbe Magic Stones from the Vubbes. {nl} And then insert them inside a gap and the Crystal Wall should open.
-QUEST_LV_0100_20150317_001231	$What are you doing here? {nl} Hurry and get into the mines. {nl} Weren't we supposed to meet in the 1st floor?
+QUEST_LV_0100_20150317_001230	$If you still want to enter the Closed Area, collect Vubbe Magic Stones from the Vubbes. {nl}And then insert them inside a gap and the Crystal Wall should open.
+QUEST_LV_0100_20150317_001231	$What are you doing here? Hurry and get into the mines. {nl}Weren't we supposed to meet in the 1st floor?
 QUEST_LV_0100_20150317_001232	$No demon will be able to pass here.{nl}I will do my best to protect this place so they can't interrupt the Paladin Master.
-QUEST_LV_0100_20150317_001233	Firm faith in us is the greatest shield.{nl}We always fought with our justice and faith.
+QUEST_LV_0100_20150317_001233	$Absolute faith is our greatest shield.{nl}In accordance with the Goddess's will, we came to fight using our justice and conviction.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1233]: Goddess -> goddess ?

File: QUEST_LV_0100.tsv [hunk 4]
```diff
@@ -1033,271 +1033,271 @@ QUEST_LV_0100_20150317_001032	$I should have requested this of you all at once.{
...
 QUEST_LV_0100_20150317_001234	{memo X}Well done.{nl}As the Paladin Master, I salute to your accomplishment.
 QUEST_LV_0100_20150317_001235	$Watcher James
 QUEST_LV_0100_20150317_001236	$We must protect this place from the demons at any cost.{nl}The land here is sacred.
 QUEST_LV_0100_20150317_001237	$I would have fled long time ago if it wasn't for that pledge.{nl}I am still the son of the great Watcher.
 QUEST_LV_0100_20150317_001238	$We have to somehow thwart these demons.{nl}I cannot let this danger pass to our brothers in the church.
 QUEST_LV_0100_20150317_001239	{memo X}We can't go in due to the barrier.{nl}To open the door, we should go through 1st floor of the basement.
-QUEST_LV_0100_20150317_001240	When I meet you again, is this world going to be peaceful.. I hope it is.
-QUEST_LV_0100_20150317_001241	I never expected those demons to come down this far.{nl}I hope our brothers can persist in their fight against them.
+QUEST_LV_0100_20150317_001240	$Will this world be peaceful when we meet again... I hope it will.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1240]: Will this world be peaceful when we meet again? 

(replace the ellipsis with a question mark?)

File: QUEST_LV_0100.tsv (hunk 1 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:1148]: @Sunaries they're two different relics.

The relic in that PR is for the Krivis Job Advancement. You save a Krivis brother who is carrying the sacred relic of the Krivis. 

In this PR, the Holy Relic is with regards to the Tenet Church.

File: QUEST_LV_0100.tsv (hunk 3 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:1233]: right xD

File: QUEST_LV_0100.tsv (hunk 4 above)
Comment (ttgmichael) [inline QUEST_LV_0100.tsv:1240]: ok :+1:

File: QUEST_LV_0100.tsv [hunk 5]
```diff
@@ -1033,271 +1033,271 @@ QUEST_LV_0100_20150317_001032	$I should have requested this of you all at once.{
...
 QUEST_LV_0100_20150317_001234	{memo X}Well done.{nl}As the Paladin Master, I salute to your accomplishment.
 QUEST_LV_0100_20150317_001235	$Watcher James
 QUEST_LV_0100_20150317_001236	$We must protect this place from the demons at any cost.{nl}The land here is sacred.
 QUEST_LV_0100_20150317_001237	$I would have fled long time ago if it wasn't for that pledge.{nl}I am still the son of the great Watcher.
 QUEST_LV_0100_20150317_001238	$We have to somehow thwart these demons.{nl}I cannot let this danger pass to our brothers in the church.
 QUEST_LV_0100_20150317_001239	{memo X}We can't go in due to the barrier.{nl}To open the door, we should go through 1st floor of the basement.
-QUEST_LV_0100_20150317_001240	When I meet you again, is this world going to be peaceful.. I hope it is.
-QUEST_LV_0100_20150317_001241	I never expected those demons to come down this far.{nl}I hope our brothers can persist in their fight against them.
+QUEST_LV_0100_20150317_001240	$Will this world be peaceful when we meet again?... I hope it will.
```
Comment (Sunaries) [inline QUEST_LV_0100.tsv:1240]: I would leave the ellipsis out.

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
Thank you :+1:

**Pull request merged**